spec/spec-patches/v17/backup_message.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/appliance_backup.yml b/appliance_backup.yml
+index ae62e93..02e36cc 100644
+--- a/appliance_backup.yml
++++ b/appliance_backup.yml
+@@ -86,6 +86,9 @@ appliances-backup-id-status:
+                   enum:
+                     - failure
+                     - success
++                message:
++                  type: string
++                  description: A free text field that shows what internally the backup is doing.
```

Comment:
The english is a bit iffy here.

Reply:
its copy paste from upstream in the controller 😜

Reply:
Yeah I will ping someone upstream :)